lighthouse-core/lib/asset-saver.js

```diff
@@ -143,11 +145,14 @@ function prepareAssets(artifacts, audits) {
 function saveAssets(artifacts, audits, pathWithBasename) {
   return prepareAssets(artifacts, audits).then(assets => {
     assets.forEach((data, index) => {
-      const traceData = data.traceData;
       const traceFilename = `${pathWithBasename}-${index}.trace.json`;
-      fs.writeFileSync(traceFilename, JSON.stringify(traceData, null, 2));
```

Comment:
I'm planning on handling the pretty printed file in WPT since I don't think its an excuse to be human-hostile here